openid-provider-commands-1_0.md

```diff
@@ -190,7 +190,11 @@ The following Claims are used within the Command Token:
 
 - **aud**  
   REQUIRED.  
-  Audience(s), as specified in Section 2 of {{OpenID.Core}}.
+  The Command Endpoint.
```

Comment:
What's about "URL of the RP's Command Endpoint", or a bit shorter "the RP's Command Endpoint"?

This URL should be the sole audience of this claim, OpenID Connect Core allows to specify multiple audiences, as an array, and that leaves some room for misuse.

Reply:
Command Endpoint is a defined term earlier.